src/ResourceManager/Sql/Commands.Sql/ImportExport/Cmdlet/NewAzureSqlDatabaseImport.cs

```diff
@@ -0,0 +1,118 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Commands.Sql.ImportExport.Model;
+
+namespace Microsoft.Azure.Commands.Sql.ImportExport.Cmdlet
+{
+    /// <summary>
+    /// Defines the AzureRmSqlDatabaseImport cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmSqlDatabaseImport", ConfirmImpact = ConfirmImpact.Low)]
+    public class NewAzureSqlDatabaseImport : ImportExportCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the name of the database to use.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "SQL Database name.")]
+        [ValidateNotNullOrEmpty]
+        public string DatabaseName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the edition of the database
+        /// </summary>
+        [Parameter(Mandatory = true, HelpMessage = "The edition of the database")]
+        [ValidateNotNull]
+        public DatabaseEdition Edition
```

Comment:
Where would this value come from?  Is there a ValidationSet that could be used, or would this come from a Get-AzureRmDatabase call?